content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/onboarding/dynamic-onboarding.md

```diff
@@ -0,0 +1,484 @@
+---
+date: '2022-11-16'
+title: "Onboarding Members to Dynamic Networks"
+menu:
+  corda-5-alpha:
+    identifier: corda-5-alpha-dynamic-onboarding
+    parent: corda-5-alpha-onboarding-tutorials
+    weight: 3000
+section_menu: corda-5-alpha
+---
+This section describes how to configure a [dynamic network](../../network-types.html#dynamic-networks) to onboard new members. It assumes you have configured the [MGM for the network](mgm-onboarding.html).
+
+## Set Variables
+Set the values of variables for use in later commands:
+
+1. Set the P2P gateway host and port and the [REST API](../../../developing/rest-api/rest-api.html) host and port. This may also vary depending on where you have deployed your cluster(s) and how you have forwarded the ports.
+   {{< tabs >}}
+   {{% tab name="Bash"%}}
+   ```shell
+   export RPC_HOST=localhost
+   export RPC_PORT=8888
+   export P2P_GATEWAY_HOST=localhost
+   export P2P_GATEWAY_PORT=8080
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
```

Comment:
I think it would be good to highlight anywhere we mention Powershell commands that these work for Powershell version 7. @yift-r3 can you confirm?

Reply:
Yes, the `-SkipCertificateCheck` won't work with Powershell 5.x (which is the default version on Windows)

Reply:
Added a general note at the start of the dynamic and MGM pages.